newIDE/app/src/EventsSheet/InstructionEditor/InstructionOrObjectSelector.js

```diff
@@ -68,6 +68,18 @@ export const styles = {
 
 export type TabName = 'objects' | 'free-instructions';
 
+const moveDeprecatedInstructionsDown = (
+  results: Array<SearchResult<EnumeratedInstructionMetadata>>
+) => {
+  const deprecatedResults = results.filter(result =>
+    result.item.fullGroupName.includes('deprecated')
```

Comment:
It will only work for the English language, right?
Translating "deprecated" would be weak too.

Reply:
Yes it will only work in english. I think we should remove the translation markers for this extension anyway.